release.py

```diff
@@ -419,16 +419,17 @@ def get_new_version_post_release(repo: github.Repository.Repository, branch_name
     latest_version = get_latest_release_tag(repo, branch_name)
 
     if latest_version is None:
-        logger.error('No version tags found in branch "{branch_name}".')
+        logger.error('No version tags found in branch %r.', branch_name)
         exit(1)
 
-    logger.info(f'Latest version in branch "{branch_name}": {latest_version}')
+    logger.info('Latest version in branch %r: %s', branch_name, latest_version)
 
     # Check if the latest version is in SEMVER, in case it has a suffix like 3.0.0.post1.
     if not re.match(r'^\d+\.\d+\.\d+$', latest_version):
         logger.error(
-            f'Latest version "{latest_version}" must be in format '
-            f'X.Y.Z. Please input the new version manually (remember to add the .dev0 suffix).'
+            'Latest version %r must be in format '
+            'X.Y.Z. Please input the new version manually (remember to add the .dev0 suffix).',
+            latest_version,
```

Comment:
Maybe this could be fixed while at it:

Instructions unclear: X.Y.Z or X.Y.Z.dev0?